samples/helloworldservlet/README.md

```diff
@@ -2,3 +2,37 @@
 A simple Servlet example that demonstrates how to use the Alexa SkillServlet. 
 
 For more information, please refer to [Developing Your First Skill](https://alexa-skills-kit-sdk-for-java.readthedocs.io/en/latest/Developing-Your-First-Skill.html) for detailed instructions.
+
+Test your skill on Tomcat server locally
+========================================
+
+Requirements
+------------
+
+1. Tomcat server. [Download and installation instructions](https://tomcat.apache.org/download-80.cgi)
+
+2. ngrok (required since alexa requires a https endpoint to send request to). [Download and install](https://ngrok.com/download)
+
+Steps
+-----
+
+- Change your current working directory to root `alexa-skills-kit-sdk-for-java/samples/helloworldservlet` and run `mvn compile war:war`. This will generate a `helloworldservlet.war` file in
+the target folder of the skill's repo.
+
+- Move the war file to webapps folder of tomcat installation directory.
```

Comment:
Do we need any details about the ROOT webapp or how to make this run on the root path? (/)

Reply:
added an external link which explains multiple options of running the app on root path.